float32 blocked_probability
float32 duration_blocked
string prediction

